B8L0TF/Json/ReadAndWriteJson.cs

```diff
@@ -0,0 +1,88 @@
+﻿using B8L0TF.Models;
+using System.Text.Json;
+
+namespace B8L0TF.Json
+{
+    internal class ReadAndWriteJson
```

Comment:
Ez az osztály nem csak olvassa és írja a JSON fájlt. Itt van exta logika is, ami mehetett volna egy külön osztályba.